packages/server/src/cli.ts

```diff
@@ -26,6 +26,7 @@ export class Entry extends Command {
 		version: flags.version({ char: "v" }),
 		"no-auth": flags.boolean({ default: false }),
 		"allow-http": flags.boolean({ default: false }),
+		password: flags.string(),
```

Comment:
We could generate the password here and add it as a default. What do you think?

Reply:
@kylecarbs for non interactive env's a flag should be provided I think

Reply:
@kylecarbs Maybe, but in this case, `code-server --help` will print always a random password as a default value for `--password` argument, won't it?

Reply:
Ah thats correct. I agree.